src/functions.js

```diff
@@ -121,7 +120,30 @@ var _ = Mavo.Functions = {
 			return n && n > 1? [arr] : arr;
 		}
 
-		return n && n > 1? arr.slice(0, n) : arr[0];
+		if (n && n > 1) {
+			tempArr = [];
```

Comment:
You forgot var/let here which creates a global. Be **very** careful of that, it can be the source of super weird bugs.

Also, nitpick: For these kinds of variables (building a return value gradually), we typically use `ret` as a convention.